src/darwin/Framework/CHIP/templates/MTRClusterNames.zapt

```diff
@@ -0,0 +1,8 @@
+{{> header excludeZapComment=true}}
```

Comment:
This should not be a generated file.  See how MTRAttributeSpecifiedCheck has a non-generated header and a generated implementation, say.

Reply:
Done.